backend/api/views.py

```diff
@@ -1,3 +1,36 @@
-from django.shortcuts import render
+from rest_framework import viewsets, mixins
+from serializers import BusinessSerializer
+from ickly import secret_settings
+from sodapy import Socrata
+
+class Business(object):
+    def __init__(self, camis, dba, cuisine_description, address):
+        self.camis = camis
+        self.dba = dba
+        self.cuisine_description = cuisine_description
+        self.address = address
+
+
+class BusinessesViewSet(mixins.ListModelMixin,
+                        viewsets.GenericViewSet):
+
+    serializer_class = BusinessSerializer
+
+    def get_queryset(self):
+        client = Socrata("data.cityofnewyork.us", secret_settings.APP_TOKEN)
```

Comment:
I got the following error at this line:

```
SSLError at /api/v1/businesses/
hostname 'data.cityofnewyork.us' doesn't match either of '*.data.socrata.com', 'data.socrata.com'
```

Seems related to the way we are using the socrata lib, some kind of misconfiguration or bad input or something. Are you able to reproduce this?